ramls/loan-policy.json

```diff
@@ -0,0 +1,173 @@
+          "type": "boolean",
+          "description": "Unlimited renewals"
+        },
+        "numberAllowed": {
+          "type": "number",
+          "description": "Number of renewals allowed"
+        },
+        "renewFromId": {
+          "type": "string",
+          "description": "Renew from date"
+        },
+        "differentPeriod": {
+          "type": "boolean",
+          "description": "Renewal period different from original loan"
+        },
+        "period": {
+          "type": "object",
+          "$ref": "period.json",
+          "description": "Alternate loan period for renewals"
+        },
+        "alternateFixedDueDateScheduleId": {
+          "type": "string",
+          "description": "Alternate fixed due date schedule (due date limit) for renewals"
+        }
+      }
+    },
+    "requestManagement": {
+      "type": "object",
+      "additionalProperties": true,
+      "properties": {
+        "recalls": {
+          "type": "object",
+          "additionalProperties": true,
+          "properties": {
+            "alternateGracePeriod": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Alternate grace period for recalled items"
+            },
+            "minimumGuaranteedLoanPeriod": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Minimum guaranteed loan period"
+            },
+            "recallReturnInterval": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Recall return interval"
+            },
+            "allowRecallsToExtendOverdueLoans": {
+              "type": "boolean",
+              "description": "Whether recalls are allowed to extend overdue loans",
+              "default": false
+            },
+            "alternateRecallReturnInterval": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Alternate recall return interval for overdue loans"
+            }
+          }
+        },
+        "holds": {
+          "type": "object",
+          "additionalProperties": true,
+          "properties": {
+            "alternateCheckoutLoanPeriod": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Alternate loan period at checkout for items with active, pending hold request"
+            },
+            "renewItemsWithRequest": {
+              "type": "boolean",
+              "description": "Allow renewal of items with active, pending hold request"
+            },
+            "alternateRenewalLoanPeriod": {
+              "type": "object",
+              "$ref": "period.json",
+              "description": "Alternate loan period at renewal for items with active, pending hold request"
+            }
+          }
+        },
+        "pages": {
+          "type": "object",
+          "additionalProperties": false,
```

Comment:
Should we allow additional properties in case of schema updating in mod-circulation?